models/classes/runner/synchronisation/action/Timeout.php

```
@@ -49,10 +50,17 @@ public function process(): array
                 $this->setOffline();
             }
 
+            $container = $this->getServiceLocator()->getContainer();
+            $isPermanentLateSubmission = $container->has(FeatureFlagChecker::class)
```

PHPMD (2.15.0)

Avoid excessively long variable names like $isPermanentLateSubmission. Keep variable name length under 20. (Naming Rules)

(LongVariable)